openreview/venue/group.py

```diff
@@ -84,15 +98,27 @@ def set_landing_page(self, group, parentGroup):
                     content = content.replace("var PARENT_GROUP_ID = '';", "var PARENT_GROUP_ID = '" + parentGroup.id + "';")
                 content = content.replace("var HEADER = {};", "var HEADER = " + json.dumps(header) + ";")
                 content = content.replace("var VENUE_LINKS = [];", "var VENUE_LINKS = " + json.dumps(links) + ";")
```

Comment:
I would like to update the GroupDirectory webfield to use the parent property so that this VENUE_LINKS var isn't needed

Reply:
Now that all the groups have a parent, can this be implemented in the openreview-web side without having to set a component for every group?

if group.web then render group.web otherwise render the group directory

Reply:
Yeah I like that idea.